react/components/ProductImages/components/ProductImage.tsx

```diff
@@ -107,6 +106,7 @@ const ProductImage: FC<Props> = ({
             alt={alt}
             title={alt}
             loading={index === 0 ? 'eager' : 'lazy'}
+            fetchPriority={index === 0 ? 'high' : undefined}
```

Comment:
```suggestion
            {...(index === 0 && { fetchPriority: 'high' })}
```

Got it, you are right about the false value! So, instead, I suggest we go for this instead of undefined

Reply:
Good, makes sense.